src/pages/landingPage/navbar/RightSection.tsx

```diff
@@ -0,0 +1,18 @@
+import SettingsOutlinedIcon from '@mui/icons-material/SettingsOutlined';
+import AppsRoundedIcon from '@mui/icons-material/AppsRounded';
+import AccountCircleOutlinedIcon from '@mui/icons-material/AccountCircleOutlined';
+
+const RightSection = () => {
+    return (
+      <div className="w-[200] h-12 gap-6">
```

Comment:
What is w-[200]?

Reply:
I thought it means setting width to 200px. But i don't know how to check if it works. I have changed it to w-52.